package.json

```diff
@@ -57,8 +57,9 @@
   },
   "dependencies": {
     "@changesets/cli": "^2.16.0",
+    "@codechecks/client": "^0.1.11",
     "@openzeppelin/contracts": "^4.3.2",
     "@openzeppelin/contracts-upgradeable": "^4.3.2",
-    "@codechecks/client": "^0.1.11"
+    "depcheck": "^1.4.2"
```

Comment:
Maybe we could do `npx depcheck` instead of installing it as a dependency here (depcheck seems to come with a lot of dependencies of its own and we try to avoid adding dependencies where possible), but that might be a bit incongruous with what we do elsewhere. cc @tynes what do you think?

Reply:
I agree that `depcheck` adds a ton of dependencies that we don't really need, for example the `yarn.lock` now has vue related dependencies... we would like to keep the attack surface as low as possible when it comes to adding in extra deps. Doing `npx depcheck` works as it will install it at runtime, it runs in CI so installing it there is fine. That would let us not need to include it in the `package.json` `dependencies`

Reply:
Good call. Just updated.